docs/csharp/how-to/compare-strings.md

```diff
@@ -68,23 +68,10 @@ and "de-DE" cultures.
 
 :::code language="csharp" interactive="try-dotnet-method" source="../../../samples/snippets/csharp/how-to/strings/CompareStrings.cs" id="Snippet3":::
 
-This sample demonstrates the operating system-dependent nature of linguistic
-comparisons. The host for the interactive window is a Linux host. The
-linguistic and ordinal comparisons produce the same results. If you
-run this same sample on a Windows host, you see the following output:
-
-```console
```

Comment:
In .NET 5+, this is no longer accurate, so I removed it here and a little further down. Windows behavior is now the same as other OSs.